src/Utilities/QuaternionHelpers.hpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines helper functions for converting between `std::array` and boost
+/// quaternions.
+
+#pragma once
+
+#include <array>
+#include <boost/math/quaternion.hpp>
+
+#include "DataStructures/DataVector.hpp"
```

Comment:
You can't use files in `DataStructures` from inside `Utilities`.  If you need it you will have to move this file elsewhere.

Reply:
Ah ok yeah. So if I am interested in just converting between DataVectors and quaternions like your above comment suggests (getting rid of the one element arrays), I should move these helper functions somewhere else.

How about just something like `Domain/FunctionsOfTime/QuaternionHelpers.hpp` because that's where I will eventually put a quaternion function of time which will make use of these helpers.

Reply:
That sounds good.